main.py

```diff
@@ -1,7 +1,65 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Actor, Genre
 
 
 def main() -> QuerySet:
-    pass
+    Genre.objects.create(
+        name="Western"
+    )
+    Genre.objects.create(
+        name="Action"
+    )
+    Genre.objects.create(
+        name="Dramma"
+    )
```

Comment:
DRY ! Look you can create list of genres and then create them in db using for loop.